crates/wash-cli/src/up/mod.rs

```diff
@@ -430,7 +430,8 @@ pub async fn handle_up(cmd: UpCommand, output_kind: OutputKind) -> Result<Comman
                 }
             }
             Err(e) => {
-                eprintln!("🟨 Couldn't download wadm {WADM_VERSION}: {e}");
+                let wadm_version: String = cmd.wadm_opts.wadm_version.clone();
+                eprintln!("🟨 Couldn't download wadm {wadm_version}: {e}");
```

Comment:
```suggestion
                eprintln!("🟨 Couldn't download wadm {}: {e}", cmd.wadm_opts.wadm_version);
```

(Response to https://github.com/wasmCloud/wasmCloud/pull/1839#discussion_r1556439390)

Not that it really matters here